server/channels/app/webhook.go

```diff
@@ -109,11 +108,14 @@ func (a *App) TriggerWebhook(c request.CTX, payload *model.OutgoingWebhookPayloa
 		contentType = "application/x-www-form-urlencoded"
 	}
 
+	var wg sync.WaitGroup
```

Comment:
Is there a possibility that this would bring some performance degradation?

Reply:
No. Because this whole thing still happens in a separate goroutine.